### PR TITLE
fix jwt subscription blocking issue

### PR DESCRIPTION
**Pull request by dushansilva**
fixes https://github.com/wso2/product-apim/issues/8433

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java [hunk 1]
```diff
@@ -10480,7 +10480,74 @@ public void addSubscriptionPolicy(SubscriptionPolicy policy) throws APIManagemen
...
+            }
+            handleException("Failed to get Subscription Block condition with condition value " + conditionValue
+                    + " of tenant " + tenantDomain, e);
+        } finally {
+            APIMgtDBUtil.closeAllConnections(selectPreparedStatement, connection, resultSet);
+        }
+        return blockCondition;
+    }
+
+    private void addGraphQLQueryAnalysisInfo(Connection conn, int maxDepth, int maxComplexity, int policyId)
```
Comment (RAVEENSR) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:10535]: Is this related to JWT subscription block fix?

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java [hunk 2]
```diff
@@ -10480,7 +10480,74 @@ public void addSubscriptionPolicy(SubscriptionPolicy policy) throws APIManagemen
                 handleException("Failed to add Subscription Policy: " + policy, e);
             }
         } finally {
-            APIMgtDBUtil.closeAllConnections(policyStatement, conn, null);
+            APIMgtDBUtil.closeAllConnections(policyStatement, conn, rs);
```
Comment (RAVEENSR) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:10483]: rs is not defined. Travis build is also failed due to this.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java (hunk 1 above)
Comment (RAVEENSR) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:10535]: Is this related to JWT subscription block fix?

Comment (dushansilva) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:10535]: nope ti think it came after i took a pull

**Comment by dushansilva** [comment]
closing this PR since there were build failures due to conflicts
